src/index.ts

```diff
@@ -71,9 +71,15 @@ const onError = (error) => {
 // event listener for "listening" event.
 const onListening = () => {
   const addr = server.address();
-  const bind = typeof addr === 'string' ? 'pipe ' + addr : 'port ' + addr.port;
-  console.log('listening on ' + bind);
-  logger.debug('listening on ' + bind);
+  if (addr) {
+    const bind =
+      typeof addr === 'string' ? 'pipe ' + addr : 'port ' + addr.port;
+    console.log('listening on ' + bind);
```

Comment:
We should stop using console.log but in current commit this is okay. I hope next time we will improve logging